generate_brick.py

```diff
@@ -215,35 +239,40 @@ def define_properties(definitions, superprop=None):
         "tags": [TAG.Location],
     },
     "Point": {
-        # "tags": [TAG.Point],
+        "tags": [TAG.Point],
```

Comment:
I wanted to have this from the beginning, but it means that we have to add `TAG.Point` to all the point classes (and better be inside `bricksrc/*.py` instead of this script for consistency imho.)

Reply:
So my suggestion is to add `TAG.Point` to all the point classes.

Reply:
We do need to add `TAG.Point` to all of the point classes, but we also need it here because it is the tag of the root `BRICK.Point` class. We can handle adding the tags in a subsequent PR where we add all of the tags to the classes

Reply:
Yeah I didn't mean to remove this one. This needs to be here anyway. Okay, let's create another PR for missing tags.